lib/features/home/home.dart

```diff
@@ -242,9 +252,11 @@ class _HomePageState extends State<HomePage> with TrayListener, WindowListener {
         MenuItem(
             key: 'exit',
             label: 'exit'.i18n,
-            onClick: (item) {
-              windowManager.destroy();
-              LanternFFI.exit();
+            onClick: (item) async {
+              await trayManager.destroy();
+              await LanternFFI.exit();
```

Comment:
We are waiting for Lantern to finish running clean up functions and immediately exiting after now